elastic/synthetics-recorder
# fix: ESLint header path for demo app

## Summary

Resolves #266.

Fixes a bug where the eslint config file failed to generate a path that enabled the script to read the file containing the MIT license copy for our file header lint rule.

## Implementation details

Previously we were using a relative path, which didn't work for the test app, as it's nested. I've modified it to use the `path` API instead.

I've also skipped the test app from our linting process, as we haven't maintained it in the past. I can unskip it and implement the necessary fixes if we'd prefer.

## How to validate this change

Run `npm install`. During the `next build` step, you should not see any errors logged:

```
Installation complete
  • electron-builder  version=23.0.3
  • loaded configuration  file=/Users/jk/git/justinkambic/synthetics-recorder/electron-builder.yml
  • public/electron.js not found. Please see https://medium.com/@kitze/%EF%B8%8F-from-react-to-an-electron-app-ready-for-production-a0468ecb1da3
  • loaded parent configuration  preset=react-cra
  • rebuilding native dependencies  dependencies=sharp@0.30.4 platform=darwin arch=x64
  • install prebuilt binary  name=sharp version=0.30.4 platform=darwin arch=x64 napi=
```
